packages/core-components/src/components/Status/icons/PendingIcon.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+
+export const PendingIcon: React.FC<{
+  className: string;
+  dataTestId: string;
+}> = ({ className, dataTestId }): React.ReactElement => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      height="24px"
+      viewBox="0 -960 960 960"
+      width="24px"
+      fill="#e8eaed"
```

Comment:
should these have width, height and fill? i am not knowledgeable enough to know for sure :) maybe it's mainly "informative" for dev purposes and then gets overridden anyway?

Reply:
you are right, it does get overridden. I have removed those now.